packages/browser/package.json

```diff
@@ -9,9 +9,9 @@
   "engines": {
     "node": ">=6"
   },
-  "main": "dist/index.js",
-  "module": "esm/index.js",
-  "types": "dist/index.d.ts",
+  "main": "build/dist/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/types/index.d.ts",
```

Comment:
```suggestion
  "types": "dist/index.d.ts",
```

Reply:
I went with `build/dist/index.d.ts` here, as it would otherwise point to the wrong path.